application/backend/src/Mail/ForgetPasswordMail.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Mail;
+
+use App\Entity\User;
+use App\Mailer\AbstractMailer;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
+use Symfony\Component\Mailer\MailerInterface;
+
+class ForgetPasswordMail extends AbstractMailer
+{
+    private $resetFrontRouting;
+
+    public function __construct(MailerInterface $mailer, string $resetFrontRouting)
+    {
+        parent::__construct($mailer);
+        $this->resetFrontRouting = $resetFrontRouting;
+    }
+
+    public function create(array $parameters = [])
+    {
+        /** @var User $user */
+        $user = $parameters['user'];
+
+        return (new TemplatedEmail())
+            ->subject('Mot de passe oublié')
```

Comment:
It would be cool to have this in translation files, like you did with the assert